python/ucxx/_lib_async/continuous_ucx_progress.py

```diff
@@ -70,3 +72,50 @@ async def _progress_task(self):
             worker.progress()
             # Give other co-routines a chance to run.
             await asyncio.sleep(0)
+
+
+class BlockingMode(ProgressTask):
+    def __init__(self, worker, event_loop, epoll_fd):
+        super().__init__(worker, event_loop)
+
+        # Creating a job that is ready straightaway but with low priority.
+        # Calling `await self.event_loop.sock_recv(self.rsock, 1)` will
+        # return when all non-IO tasks are finished.
+        # See <https://stackoverflow.com/a/48491563>.
```

Comment:
Just a reminder that this approach (as mentioned in that updated answer) unfortunately doesn't work :(. I tried to figure out a way last time we noticed this, but didn't manage to.

Reply:
Indeed, see [my reply to the "main" review comment](https://github.com/rapidsai/ucxx/pull/116#issuecomment-1795336173).